soccer-records-model/src/main/java/cz/muni/fi/pa165/user/UserRepositoryImpl.java

```diff
@@ -55,9 +42,21 @@ public User findUserByEmail(final String email)
     }
 
     @Override
-    public User findUserById(UUID id)
+    public User getUserById(final UUID userId)
     {
-        return em.find(User.class, id);
+        if (userId == null) {
+            throw new IllegalArgumentException("Cannot search for a null id");
+        }
```

Comment:
assert maybe? It's shorter. (Conceptually one should be validated by type system.)